tests/test_form.py

```diff
@@ -109,6 +112,20 @@ def test_get_metadata(self):
         expected = u'<!-- @sky: blue -->\n<!-- @earth: None -->\n'
         self.assertEqual(actual, expected)
 
+    def test_normalize_metadata(self):
+        """Avoid some type of strings."""
+        cases = [('blue sky -->', 'blue sky'),
+                 ('blue sky ---->>', 'blue sky'),
+                 ('', ''),
+                 ('blue sky ', 'blue sky'),
+                 ('bad_bird <script>', ''),
+                 ('bad_bird <script-->>', ''),
+                 ('a' * 300, ''),
```

Comment:
well if i can't rely on this site to do weird string math anymore, what's even the point

Reply:
heh. This generates a string of 300 `a` characters  ;) 
It is to test the length :)

Reply:
:))